vllm/sampling_params.py

```diff
@@ -237,6 +238,14 @@ def sampling_type(self) -> SamplingType:
             return SamplingType.RANDOM_SEED
         return SamplingType.RANDOM
 
+    def copy(self) -> "SamplingParams":
+        """ Deep copy excluding LogitsProcessor objects"""
```

Comment:
let's explain why we exclude it? also remove leading space

Reply:
Done